repositories/postgres/api_keys.go

```diff
@@ -33,8 +20,13 @@ func (db *Database) GetApiKeyByKey(ctx context.Context, key string) (models.ApiK
 	`
 
 	var apiKey dbmodels.DBApiKey
-	err := db.pool.QueryRow(ctx, query, key).Scan(&apiKey.Id, &apiKey.OrganizationId,
-		&apiKey.Hash, &apiKey.Description, &apiKey.Role)
+	err := db.pool.QueryRow(ctx, query, key).Scan(
+		&apiKey.Id,
+		&apiKey.OrganizationId,
+		&apiKey.Key,
+		&apiKey.Description,
+		&apiKey.Role,
+	)
```

Comment:
IIUC, for now we store the hash but since it's nneither used, we still do not need it here

Reply:
yes correct.
I could have done it in the same PR, but it's mergeable as it is so let's do it more simply in a follow-up PR